src/questions/intro-to-defi.yaml

```diff
@@ -1,19 +1,354 @@
+  hint: noHint
+  explanation: Interoperability of DeFi refers to interconnection between various dapps built by developers in the DeFi ecosystem.
+  answerKeys:
+    - C
+  difficultyLevel: "Medium"
+  choices:
+    - content: DeFi is accessible from anywhere
+      key: A
+    - content: DeFi is free to use for everyone
+      key: B
+    - content: Developers can develop DeFi applications that are interconnected
+      key: C
+    - content: None of the above
+      key: D
+- uuid: 3aef2590-7787-4761-8a49-14d35c22ad88
+  type: SingleChoice
+  content: Only users can have addresses in DeFi
+  hint: Two types of address in DeFi
+  explanation: Users and Smart contracts, both have unique ids used to access them called an address.  
+  answerKeys:
+    - B
+  difficultyLevel: "Low"
+  choices:
+    - content: "True"
+      key: A
+    - content: "False"
+      key: B
+- uuid: 8f04b67d-6729-4401-80e1-9dca376765db
+  type: MultipleChoice
+  content: How is DeFi more inclusive as compared to TradFi?
+  hint: DeFi has no hidden charges, and it does not depend on credibility.
+  explanation: In DeFi there are no charges for transactions, the fund transfer is fast and affordable and does not depend on credibility of borrower and allows flash loans to its users as well.
+  answerKeys:
+    - A
+    - B
+    - C
+  difficultyLevel: "Medium"
+  choices:
+    - content: DeFi is borderless
+      key: A
+    - content: Fast and affordable money transfer
+      key: B
+    - content: No background checks are needed 
+      key: C
+    - content: Easy money-making opportunity
+      key: D  
+- uuid: 924dd82c-e5f2-48d1-ad71-ff507e978c2d
+  type: MultipleChoice
+  content: Which of the following problems related to bank loans does DeFi solve?
+  hint: Benefits of DeFi over TradFi
+  explanation: TradFi requires a lot of collateral before the loan is granted and loans are granted only to potential borrowers with high net worth/ high credibility.
+  answerKeys:
+    - A
+    - C
+  difficultyLevel: "Medium"
+  choices:
+    - content: Unpaid Centralized loans lead to loss for banks
+      key: A
+    - content: Bank loans are easy to get
+      key: B
+    - content: Banks differentiate between potential borrowers based on the credit score
+      key: C
+    - content: Banks charge low-interest rates
+      key: D
+- uuid: 078e0e8c-4da2-4db9-87d9-7d70bced38cf
+  type: MultipleChoice
+  content: How can someone access the world of DeFi?
+  hint: NoHint
+  explanation: Decentralized Finance can be accessed using the internet, decentralised applications, and a blockchain address.
+  answerKeys:
+    - B
+    - C
+    - D
+  difficultyLevel: "Low"
+  choices:
+    - content: Using Bank accounts
+      key: A
+    - content: Using Decentralized Applications
+      key: B
+    - content: Internet access
+      key: C
+    - content: Blockchain address
+      key: D
+- uuid: 89ed030b-7eb1-4054-8627-4124bf72641a
```

Comment:
Main issue is lack of global infra structure. Like currently you can send the money very easily to anyone in India.  What if you want to send to Kenya, Nigeria or Nepal (like in our case). Global transfers are a breeze in blockchain and are pain in TradFi.

Please update the question to a single choice question based on above explanation

Reply:
I have kept it a multi choice question but added valuable options instead, let me know if this is fine or i will change it to single correct and alter the options